league_rpc_linux/__main__.py

```diff
@@ -1,6 +1,8 @@
 import argparse
 import sys
 import time
+from multiprocessing import Process
+import nest_asyncio
```

Comment:
This is pretty cool.. this is what i got stuck at, constantly got "RuntimeError: This event loop is already running" Cool that you found this!